packages/cli/src/blueprints/CONTRIBUTING.md

```diff
@@ -66,24 +66,37 @@ Here are some guidelines to help you and everyone else.
 
 <details>
 
-<summary>Publish packages (for admins)</summary>
+<summary>Add changeset to pull request</code></summary>
 
-1. Generate a [personal access token](https://github.com/settings/tokens/) in GitHub, with default values for scopes (none selected).
+1. To record how a pull request affects packages, you will want to add a changeset.
 
-1. Run the `changelog` script. This generates a text that you can add to `CHANGELOG.md`.
+    The changeset provides a summary of the code change. It also describes how package versions should be updated (major, minor, or patch) as a result of the code change.
 
     ```sh
-    GITHUB_AUTH=<YOUR_PERSONAL_ACCESS_TOKEN> pnpm changelog
+    pnpm changeset
     ```
 
-1. The package follows [semantic versioning](https://semver.org/). Update the version in `package.json` accordingly.
+</details>
+
+
+<details>
+
+<summary>Publish package (for admins)</summary>
+
+1. Generate a [personal access token](https://github.com/settings/tokens/) in GitHub, with `repo` and `read:user` scopes enabled.
```

Comment:
For public repos, one can use the default scopes (none selected). For private repos, however, they will need `repo` and `read:user` scopes.

To simplify the instructions, we'll ask maintainers to select the `repo` and `read:user` scopes.